src/libstd/keyword_docs.rs

```diff
@@ -1578,12 +1578,144 @@ mod type_keyword {}
 
 #[doc(keyword = "unsafe")]
 //
-/// Code or interfaces whose [memory safety] cannot be verified by the type system.
+/// Code or interfaces whose [memory safety] cannot be verified by the type
+/// system.
```

Comment:
I think one crucial point that needs to get across is [the one discussed here](https://doc.rust-lang.org/nightly/nomicon/safe-unsafe-meaning.html):

> No matter what, Safe Rust can't cause Undefined Behavior.

In other words, I think this should at least briefly bring up the issue of [soundness](https://rust-lang.github.io/unsafe-code-guidelines/glossary.html#soundness-of-code--of-a-library).

Reply:
I added a `Why do we need a keyword for this ?` section, I'm checking everything compiles (notably the links) on my side and I'll push.